### ISSUE TITLE
Invalid imput example

### ISSUE THREAD

**Issue by LaosPac** [csound/manual#620]
<CsoundSynthesizer>
<CsOptions>
-odac
</CsOptions>
<CsInstruments>
sr = 44100
ksmps = 32
nchnls = 2
0dbfs = 1
//wave form with harmonics 1, 11 and 22
giHarmonics ftgen 0, 0, 8192, 9, 1,.1,0, 11,.1,0, 22,1,0
instr 1
asig poscil .1, p4 out asig, asig
endin
instr 2
asig poscil .2, p4, giHarmonics out asig, asig
endin
</CsInstruments>
<CsScore>
i 1 0 2 1000 ;1000 Hz sine
i 1 3 2 43100 ;43100 Hz sine sounds like 1000 Hz because of aliasing
i 2 6 4 1990 ;1990 Hz with harmonics 1, 11 and 22
             ;results in 1990*22=43780 Hz so aliased 320 Hz
             ;for the highest harmonic
</CsScore>
</CsoundSynthesizer>

This example from the manual gives this error message... why?

WARNING: could not open library '/Library/Frameworks/CsoundLib64.framework/Versions/6.0/Resources/Opcodes64/librtjack.dylib' (-1)
rtaudio: PortAudio module enabled ...
using blocking interface
error: syntax error, unexpected T_OPCODE0, expecting NEWLINE or ','  (token "out") from file /private/var/folders/gy/gs9dgh857p19nby3qgvyzmgc0000gn/T/csound-tmpjiEuRuND.csd (1), line 13:
 >>> asig poscil .1, p4 out <<<
Parsing failed due to invalid input!
Stopping on parser failure
cannot compile orchestra

thank you for help (this is for a whole school class)

we have Mac computers. Happens with Big Sur and Monterey . The guy with windows has not these problems

**Comment by LaosPac** [comment]
we are using csound QT. just installed. Sorry for ignorance but we don't understand your answer. please could you give us a practical help to make it work for the class

**Comment by vlazzarini** [comment]
This is not a bug, we can't use the issue tracker for user errors. Please sign up to the Csound mailing list and ask questions there.
I'm closing this otherwise we get sidetracked.

**Comment by LaosPac** [comment]
should I delete this chat?

**Comment by LaosPac** [comment]
thanks for fast replies btw

**Comment by vlazzarini** [comment]
no need to do anything here.

**Comment by joachimheintz** [comment]
have a llok at
CsoundQt Configuration > Editor > Line Ending Mode
